docs/xenserver-automated-certification-kit-guide.md

```diff
@@ -639,27 +639,27 @@ Examples:
 1.Run only Crash Dump test class
 
 ```
-   # python ack_cli.py -n network.conf –o \
+   # ./ack_cli.py -n network.conf –o \
    "run_classes=operations_tests.CrashDumpTestClass"  
 ```
 2.Run only Multicast test class
 
 ```
-   # python ack_cli.py -n network.conf –o \
+   # ./ack_cli.py -n network.conf –o \

```

Comment:
This " –o" seems different from that in line 654 "-o"
One is long "–“， one is short "-". 
Please check.

Reply:
I have updated line 642 and line 648 to short "-".